docs/api/api-spec.md

```diff
@@ -150,10 +150,36 @@ The 'system' channels include a 'default' channel which serves as the backwards
 ### Joining Channels
 Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel, except if the channel joined is the 'default'.
 
-When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  It is up to the desktop agent to determine the behavior of the 'default' channel, and if context is automaticaly broadcast over it or not.
+When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  
+The `default` channel has special behavior that makes it different from other channels.  See a detailed description [below](#default-channel-behavior).
```

Comment:
The special behaviour of the `default` channel should also be documented clearly in the API documentation, not only in the spec, as people are more likely to read the docs than the whole spec.

The spec doesn't clarify if the `default` channel is returned by `getSystemChannels`?

Reply:
Agreed about updates to the API docs - will do.

Re: `getSystemChannels` - there is definitely a gap here.   There are arguments either way - would be great to get the opinions of others who have already built or are using channel linking today.